plugins/modules/ios_interfaces.py

```diff
@@ -90,6 +82,7 @@
     - deleted
     - rendered
     - gathered
+    - purged
```

Comment:
Could you also explain the difference between purged and deleted briefly for this module?

Reply:
The `deleted` state would delete the attributes that are configured per interface or revert the interfaces to its default config. Whereas, the `purged` state is there to remove the logical/virtual interfaces that may be present.
Purged example - 
```
- name: "Purge given interfaces (Note: This will delete the interface itself)"
  cisco.ios.ios_interfaces:
    config:
    - name: Loopback888
    - name: Port-channel10
    state: purged

# "commands": [
#       "no interface Loopback888",
#       "no interface Port-channel10",
#     ],
```
Deleted example -
```
- name: "Delete module attributes of given interfaces (Note: This won't delete the interface itself)"
  cisco.ios.ios_interfaces:
    config:
    - name: GigabitEthernet0/2
    state: deleted

# "commands": [
#       "interface GigabitEthernet0/2",
#       "no description dummy desc",
#       "no duplex auto",
#       "no speed 1000",
#     ],
```
Regards